pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -7,7 +7,7 @@
 , autoconf213, which, m4
 , writeScript, xidel, common-updater-scripts, coreutils, gnused, gnugrep, curl
 , enableGTK3 ? false, gtk3, wrapGAppsHook
-, debugBuild ? false
+, debugBuild ? false, config
```

Comment:
Okay, sure. Should I do a fixup commit, rebase -i, and force push? Or just add more commits?